ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -500,4 +506,19 @@ - (void)requestTokenByCode:(NSString *)code
               completion:completionBlock];
 }
 
+- (void)tryRefreshToken:(ADAuthenticationCallback)completionBlock
+{
+    [[ADTelemetry sharedInstance] startEvent:[self telemetryRequestId] eventName:AD_TELEMETRY_EVENT_ACQUIRE_TOKEN_SILENT];
+    ADAcquireTokenSilentHandler* request = [ADAcquireTokenSilentHandler requestWithParams:_requestParams];
+    [request acquireTokenByRefreshToken:_refreshToken
+                              cacheItem:[ADTokenCacheItem new]
+                        completionBlock:^(ADAuthenticationResult *result)
+     {
+         ADTelemetryAPIEvent* event = [[ADTelemetryAPIEvent alloc] initWithName:AD_TELEMETRY_EVENT_ACQUIRE_TOKEN_SILENT
```

Comment:
Nit: should we set api ID for this telemetry event? If not, why is it necessary?